pkg/crc/preflight/preflight_checks_linux.go

```diff
@@ -157,6 +169,14 @@ func fixLibvirtEnabled() (bool, error) {
 }
 
 func checkUserPartOfLibvirtGroup() (bool, error) {
+	return checkUserPartOfGroup("libvirt")
+}
+
+func checkUserPartOfLibvirtdGroup() (bool, error) {
+	return checkUserPartOfGroup("libvirtd")
+}
+
```

Comment:
I suspect these one liner helpers could be avoided by using lambdas in the corresponding `preflightCheckAndFix` calls (but I'm really not familiar enough with go to be fully sure this would work ;)

Reply:
I prefered to use `func` as lambda-like approach, but it is not easy to understand by everybody. Keeping it readable is more important, plus it is now understandable:

`check user part of group libvirtd` 

`checkUserPartOfGroup("libvirtd")`